app/frontend/src/components/views/workspace-form/WorkSpaceForm.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+import PageTitle from '../../layouts/base/PageTitle';
+import labeled from '../../UI/labeled/Labeled';
+import Input from '../../UI/input/Input';
+import Button, { StyleType } from '../../UI/button/Button';
+import { createProjectFx } from 'store/projects';
+import TeammateList from '../../UI/teammate-list/TeammateList';
+import { Link } from 'react-router-dom';
+import { UiComponentText } from 'styles/Mixins';
+
+/**
+ * WorkspaceForm component props model
+ */
+interface Props { }
+
+/**
+ * WorkspaceForm component
+ */
+const WorkspaceForm: React.FC<Props> = () => {
+  const [title, setTitle] = useState('');
+
+  const submit = async (): Promise<void> => {
+    await createProjectFx({
```

Comment:
This effect calls function that will create a project with specified title on server. You should create a separate function for handling workspace updates. And probably you'll need to store workspace data in effector store to be able to update it dynamically.

Reply:
I'll fix it soon

Reply:
you can reach me if you need any help